python/tests/test_variable.py

```diff
@@ -85,14 +85,30 @@ def test_create_with_variances_from_numpy_1d():
     np.testing.assert_array_equal(var.variances, np.arange(4, 8))
 
 
-def test_create_scalar():
-    var = sc.Variable(1.2)
-    assert var.value == 1.2
+@pytest.mark.parametrize(
+    "value",
+    [1.2, np.float64(1.2), sc.Variable(1.2).value])
```

Comment:
What does the third case test? Looks like a duplicate of the first one.

Reply:
Paranoia maybe? I am not sure what exactly this returns (is it really the same as a Python float?), so better test.

Reply:
But 
```.py
x = Variable(1.2)
y = Variable(x.value)
assert x.dtype == float64
assert y.dtype == float64
```
Is the second assert not guaranteed because y inherits whatever x has? So doesn't the test with `sc.Variable(1.2).value` as input do the same as the test with `1.2`?

Reply:
`1.2` is a Python float. `sc.Variable(1.2).value` seems to behave exactly like a Python float, but do we really know that it is one? I am trying to ensure that disabling the conversion of the value argument in the bindings did not break anything.

Reply:
I see, never mind then.